docs/types/ambient/variables.md

```diff
@@ -2,36 +2,36 @@
 For example to tell TypeScript about the [`process` variable](https://nodejs.org/api/process.html) you *can* do:
 
 ```ts
-declare var process:any;
+declare var process: any;
 ```
 
-> You don't *need* to do this for `process` as there is already a [community maintained `node.d.ts`](https://github.com/borisyankov/DefinitelyTyped/blob/master/node/node.d.ts)
+> You don't *need* to do this for `process` as there is already a [community maintained `node.d.ts`](https://github.com/borisyankov/DefinitelyTyped/blob/master/node/node.d.ts).
 
 This allows you to use the `process` variable without TypeScript complaining:
 
 ```ts
-process.exit()
+process.exit();
 ```
 
-We recommend using an interface wherever possible e.g:
+We recommend using an interface wherever possible e.g.:
 
 ```ts
 interface Process {
-    exit(code?:number):void;
-}
+    exit(code?: number): void;
+};
 declare var process: Process;
 ```
 
 This allows other people to *extend* the nature of these global variables while still telling TypeScript about such modifications. E.g. consider the following case where we add an `exitWithLogging` function to process for our amusement:
 
 ```ts
 interface Process {
-    exitWithLogging(code?:number):void;
-}
+    exitWithLogging(code?: number): void;
+};
```

Comment:
I don't put semicolons after interfaces. Just like I don't do it for other `}` ending things e.g. `function foo(){}` or `class Foo{}` etc :rose: